lib/ansible/modules/remote_management/redfish/idrac_redfish_config.py

```diff
@@ -49,15 +49,22 @@
   manager_attribute_name:
     required: false
     description:
-      - name of iDRAC attribute to update
+      - (deprecated) name of iDRAC attribute to update
     default: 'null'
     type: str
   manager_attribute_value:
     required: false
     description:
-      - value of iDRAC attribute to update
+      - (deprecated) value of iDRAC attribute to update
     default: 'null'
     type: str
+  manager_attributes:
+    required: false
+    description:
+      - dictionary of iDRAC attribute name and value pairs to update
+    default: {}
+    type: 'dict'
+    version_added: '2.9'
```

Comment:
This should be 2.10 (the version where this new option will be added).

Reply:
Fixed